cpp/modmesh/view/RManager.cpp

```diff
@@ -214,7 +224,6 @@ void RManager::setUpMenu()
         m_appMenu = m_mainWindow->menuBar()->addMenu(QString("App"));
 
         this->addApplication(QString("sample_mesh"));
-        this->addApplication(QString("euler1d"));
```

Comment:
Since we have `Euler` in toolbar therefore removing euler1d from `App`.

Reply:
sweet